libs/client/comments/src/lib/comments.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { CommentForm, CommentKind } from '@dragonfish/shared/models/comments';
+import { ContentViewQuery, ContentViewService } from '@dragonfish/client/repository/content-view';
+import { AlertsService } from '@dragonfish/client/alerts';
+
+@Component({
+    selector: 'dragonfish-comments',
+    templateUrl: './comments.component.html',
+    styleUrls: ['./comments.component.scss'],
+})
+export class CommentsComponent {
+    @Input() itemId!: string; // The ID of the content
+    @Input() kind!: CommentKind; // The kind of comments thread it is, between all content
+    @Input() pageNum!: number; // The requested page number
+    @Output() emitPageChange = new EventEmitter<number>(); // Emits the current page number
+
+    newCommentForm = new FormGroup({
+        body: new FormControl('', [Validators.required, Validators.minLength(10)]),
```

Comment:
Did we want the minimum number of characters to be that high? I wouldn't disagree, but just noting this removes comments like "Neat" and "Meh".